src/ceph_mgr.cc

```diff
@@ -14,6 +14,7 @@
  *
  */
 
+#define PY_SSIZE_T_CLEAN
```

Comment:
we don't need this if this macro is already defined in the cmake script.

Reply:
Sorry, I see it now. In src/mgr/CMakeLists.txt, the src/ceph_mgr.cc is added to the list with the whole path ${CMAKE_SOURCE_DIR}/src/ceph_mgr.cc.